awesomplete.js

```diff
@@ -42,15 +43,18 @@ var _ = function (input, o) {
 
 	this.ul = $.create("ul", {
 		hidden: "hidden",
+        role: "listbox",
+        id: "awescomplete_list",
```

Comment:
Yes that's true, we ran into this problem in the final testing of - https://github.com/allies-computing/accessible-address-autocomplete

We solved it there by incrementing a count of instances and using that with the ID naming.

Will update this evening on my fork.